ext/pdo_dblib/dblib_driver.c

```diff
@@ -395,6 +412,8 @@ static int pdo_dblib_handle_factory(pdo_dbh_t *dbh, zval *driver_options)
 	H->assume_national_character_set_strings = 0;
 	H->stringify_uniqueidentifier = 0;
 	H->skip_empty_rowsets = 0;
+	H->datetime_convert = 0;
+	H->datetime_format = zend_string_init("", 0, 0);
```

Comment:
Default to NULL so you don't do a needless alloc.

Reply:
OK, datetime_format is now NULL unless an non-zero-length string is passed